quesma/quesma/config/config_v2.go

```diff
@@ -407,20 +407,20 @@ func (c *QuesmaNewConfiguration) TranslateToLegacyConfig() QuesmaConfiguration {
 		}
 	}
 	// Now determine Quesma final state with following heuristic
-	// if 2 pipelines with noop processor -> switch to proxy mode, ditch the whole config
-	// if one query, one ingest pipeline with noop processor -> switch to "dual-write-query-clickhouse" mode
+	// if 2 pipelines with noop processor -> switch to transparent proxy mode, ditch the whole config
+	// if one query, one ingest pipeline with noop processor -> switch to "Quesma" mode
```

Comment:
I think right now the whole logic is as simple as "if all configured processors are of type noop -> switch to transparent proxy mode"  :)

Reply:
Fixed in https://github.com/QuesmaOrg/quesma/pull/726/commits/87b5ea91fdd00d86b1c54d2ad327addb1c86e1e4